custom/inddex/reports/utils.py

```diff
@@ -56,7 +56,7 @@ def _format_val(val):
     if isinstance(val, int):
         return str(val)
     if isinstance(val, float):
-        return str(int(val)) if val.is_integer() else str(val)
+        return f"{val:.5g}"
```

Comment:
Sweet

Reply:
Yeah - I wasn't happy about arbitrarily truncating at two decimal places.  However, this means I had to tediously go through the test data and update a bunch of numbers.